manifest.ini

```diff
@@ -3,26 +3,26 @@ server_url = https://updates-autopush.stage.mozaws.net
 push_server = wss://autopush.stage.mozaws.net
 router = stage.autopush.routerv2
 storage = stage.autopush.storage
-bugzilla_api = https://bugzilla-dev.allizom.org
-bugzilla_user = chartjes@mozilla.com
-bugzilla_api_key = oelpF8y4nHYuYf3xOxll4XQXTJtPAkIwrs1BmHvO 
+bugzilla_api =
```

Comment:
don't you still require these for the test? if so, you could invoke them as ENV vars like so: https://github.com/mozilla-services/autograph-loadtests/blob/master/loadtest.py#L21-L26
then you can just set those values locally in your bash_profile and later, on Jenkins, we'll just add them as ENV vars as well

Reply:
also, if these have been removed from the project, there's prob no reason to keep these empty keys here any longer

Reply:
I am going to be using them but just not yet -- still experimenting with changes to the BugZilla posting library I will be using. I wanted to keep them in as a place-holder